packages/plugin-ext/src/hosted/node/plugin-reader.ts

```diff
@@ -54,6 +59,18 @@ export class HostedPluginReader implements BackendApplicationContribution {
                     res.status(404).send(`No such file for plugin with id '${escape_html(pluginId)}'.`);
```

Comment:
Strange, this code look so old, on master: https://github.com/theia-ide/theia/blob/master/packages/plugin-ext/src/hosted/node/plugin-reader.ts#L51-L71

Reply:
I will rebase my changes, thanks.